errors.js

```diff
@@ -165,12 +165,6 @@ const ErrPermissionUpdateUsername = new APIError('You do not have permission to
   status: 403
 });
 
-// ErrSameUsernameProvided is returned when attempting to update a username to the same username.
-const ErrSameUsernameProvided = new APIError('Same username provided.', {
```

Comment:
We don't enforce this on the server side? @wyattjoh

Reply:
We do!

https://github.com/coralproject/talk/pull/1153/files#diff-79a167f71b013e1de67ca4450084bc9cR283